setup.md

```diff
@@ -27,16 +27,24 @@ In the lesson, you will find out how to access the data in this folder.
 > or [(KDE) Konsole](https://konsole.kde.org/)
 > or [xterm](https://en.wikipedia.org/wiki/Xterm),
 > which can be found via the applications menu or the search bar.
-> If your machine is set up to use something other than bash, you can run it by opening a terminal and typing `bash`.
+> 
+> To check if your machine is set up to use something other than Bash, type `echo $SHELL` in your terminal window.
+>
+> If your machine is set up to use something other than Bash, you can run it by opening a terminal and typing `bash`.
 >
 > ### macOS
-> For a Mac computer, the default Unix Shell is Bash,
-> and it is available via the Terminal Utilities program within your Applications folder.
+> For a Mac computer running Mac OS Mojave or earlier releases, the default Unix Shell is Bash.
+> For a Mac computer running Mac OS Catalina or later releases, the default Unix Shell is Zsh.
+> Your default shell is available via the Terminal program within your Utilities folder.
 >
 > To open Terminal, try one or both of the following:
-> * Go to your Applications. Within Applications, open the Utilities folder. Locate Terminal in the Utilities folder and open it.
+> * In Finder, select the Go menu, then select Utilities. Locate Terminal in the Utilities folder and open it.
 > * Use the Mac ‘Spotlight’ computer search function. Search for: `Terminal` and press <kbd>Return</kbd>.
 >
+> To check if your machine is set up to use something other than Bash, type `echo $SHELL` in your terminal window.
+>
+> If your machine is set up to use something other than Bash, you can run it by opening a terminal and typing `bash`.
+>
```

Comment:
This section is a duplication of the section above, is that intentional?

Reply:
My assumption was that Linux users would use the Linux section, and macOS users would  use the macOS section. Though the instruction could be factored out and placed at the end, it might be more trouble to indicate that the instruction applies to both Linux and macOS users.

Reply:
While true, I thought we [agreed that this PR would address Macs only ](https://github.com/swcarpentry/shell-novice/pull/1049#issuecomment-547906483)

Reply:
Correct, and I may have mixed up a couple of separate but related issues. The current setup.md for Linux users tells them to type `bash` if their default shell isn't set to Bash, but doesn't explain how to check what their default shell is.

Though it's quite unlikely a Linux system will use something other than Bash, I wasn't comfortable with removing existing instructions.